Mathlib/Analysis/Convex/StrictConvexBetween.lean

```diff
@@ -16,10 +16,25 @@ space.
 
 -/
 
+open Metric
+open scoped Convex
 
+variable {V P : Type*} [NormedAddCommGroup V] [NormedSpace ℝ V]
+
+section to_move
 variable {V P : Type*} [NormedAddCommGroup V] [NormedSpace ℝ V] [PseudoMetricSpace P]
+  [NormedAddTorsor V P]
+
+-- TODO: Is there a better place for this lemma?
+lemma Wbtw.dist_add_dist {x y z : P} (h : Wbtw ℝ x y z) : dist x y + dist y z = dist x z := by
+  obtain ⟨a, ⟨ha₀, ha₁⟩, rfl⟩ := h; simp [abs_of_nonneg, ha₀, ha₁, sub_mul]
```

Comment:
This lemma is now not used, right? Can you remove it?

Reply:
I could, but I think it's important to have, no?

Reply:
Ah, I missed that this only needed `PseudoMetricSpace`.

Can you add a docstring that references the `iff`, perhaps
```suggestion
-- TODO: Is there a better place for this lemma?
/-- In `MetricSpace`, this is true as an `Iff`,`dist_add_dist_eq_iff`. -/
lemma Wbtw.dist_add_dist {x y z : P} (h : Wbtw ℝ x y z) : dist x y + dist y z = dist x z := by
  obtain ⟨a, ⟨ha₀, ha₁⟩, rfl⟩ := h; simp [abs_of_nonneg, ha₀, ha₁, sub_mul]
```

Reply:
It's not `PseudoMetricSpace` that's important. It's `StrictConvexSpace`!